projects/canviewer/canviewer.py

```diff
@@ -0,0 +1,105 @@
+"""
+Callback function when receiving a CAN message, launch and update GUI
+"""
+from canserver import init_can
+from gui.gui import Window
+from utils import *
```

Comment:
Programming heresy with the wildcard import, I know, but it's a file we control and allows the user who's adding a processing function to just define it in utils and set in in the YAML without having to touch this code